scripts/check-setup.sh

```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#!/usr/bin/env sh
```

Comment:
I think this one should be converted to bash as well.  Most of us are just so used to /bin/sh being bash that we forget, but thats not always the case.